CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- Fixed `POST /collections/test-collection/items` returning an item with an empty links array.
```

Comment:
Can you leave a link to the pr - I was going to ask you for the other pr too. Looks great otherwise